src/types/DSNP.d.ts

```diff
@@ -1,132 +1,137 @@
 import { EthereumAddress } from "./Strings";
 
-export declare type DSNPMessage =
```

Comment:
Turns out this doesn't work.  It is only ever a type and can only be used with type literals.